graylog2-web-interface/src/views/components/contexts/DashboardPageContextProvider.tsx

```diff
@@ -84,15 +84,17 @@ const DashboardPageContextProvider = ({ children }: { children: React.ReactNode
     history.replace(newUri);
   }, [history, query]);
 
-  const setDashboardPageParam = (nextPage) => updatePageParams(nextPage);
-  const unSetDashboardPageParam = () => updatePageParams(undefined);
+  const setDashboardPageParam = useCallback((nextPage) => updatePageParams(nextPage), [updatePageParams]);
+  const unSetDashboardPageParam = useCallback(() => updatePageParams(undefined), [updatePageParams]);
+
+  const dashboardPageContextValue = useMemo(() => ({
+    setDashboardPage: setDashboardPageParam,
```

Comment:
We can save the `useCallback` functions before by defining these functions in this memoization.